x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/FollowerFailOverIT.java

```diff
@@ -220,4 +230,54 @@ public void testAddNewReplicasOnFollower() throws Exception {
         pauseFollow("follower-index");
     }
 
+    public void testReadRequestsReturnsLatestMappingVersion() throws Exception {
```

Comment:
Can you add a comment to each step in this test and describe why it is required? I think I understand it, but it in any way it is great to have that as documentation.

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/37633/commits/64aa5a395f9951427ee1ab097587a15a69d6cbb4